### DISCUSSION TITLE
Firebase phone Auth

### DISCUSSION THREAD

**Discussion by beekboss** [serverpod/serverpod#2910] [Q&A]
hi
I use phone auth , the problem is:
I- always get print from onFailure , event user registered successfully in firebase
2- app down with black screen with this message : "[ERROR:flutter/shell/platform/darwin/ios/framework/Source/FlutterViewController.mm(1885)] Failed to handle route information in Flutter."

any help ?

one more thing , is there any way to override the phone sign in screen ? like default country code or restrict countries ??

**Comment by beekboss** [comment]
signing in debugging : 
CALL: serverpod_auth.status.signOut duration: 39ms numQueries: 1 authenticatedUser: null
DEBUG: Firebase authenticate
DEBUG: Created Auth object
DEBUG: Verified idToken
DEBUG: Got email: null
DEBUG: Got userIdentifier: YyUtgUJjEdSy7fnrDDa3vZNiocs1
success sign in , but message : "[ERROR:flutter/shell/platform/darwin/ios/framework/Source/FlutterViewController.mm(1885)] Failed to handle route information in Flutter." when trying to return back to otp page after captcha , result failed .

**Reply by beekboss** [reply]
any help?!

**Comment by beekboss** [comment]
actually signed in success in firebase , but serverpod result fail , then black screen because of route , if restart app , will find signed in successfully .

**Comment by AlainDevs** [comment]
I have the same issue as you. Have you found any solutions yet?
